docs/topics/auth/passwords.txt

```diff
@@ -140,17 +140,18 @@ algorithm.
 
 .. admonition:: Password truncation with BCryptPasswordHasher
 
-    The designers of bcrypt truncate all passwords at 72 characters which means
-    that ``bcrypt(password_with_100_chars) == bcrypt(password_with_100_chars[:72])``.
-    The original ``BCryptPasswordHasher`` does not have any special handling and
-    thus is also subject to this hidden password length limit.
-    ``BCryptSHA256PasswordHasher`` fixes this by first hashing the
-    password using sha256. This prevents the password truncation and so should
-    be preferred over the ``BCryptPasswordHasher``. The practical ramification
-    of this truncation is pretty marginal as the average user does not have a
-    password greater than 72 characters in length and even being truncated at 72
-    the compute powered required to brute force bcrypt in any useful amount of
-    time is still astronomical. Nonetheless, we recommend you use
+    The designers of bcrypt truncate all passwords at 72 bytes which means that
+    ``bcrypt(password_with_100_bytes) == bcrypt(password_with_100_bytes[:72])``.
+    The original ``BCryptPasswordHasher`` does not have any special handling
+    and thus is also subject to this hidden password length limit.
+    ``BCryptSHA256PasswordHasher`` fixes this by first hashing the password
+    using sha256. This prevents the password truncation and so should be
+    preferred over the ``BCryptPasswordHasher``. The practical ramification of
+    the truncation is moderate. While the average user has ASCII passwords
+    which are truncated to 72 characters, passwords will all Unicode characters
```

Comment:
> passwords will all Unicode characters

Is this a typo? Should it be: "passwords with all Unicode characters"? (will -> with)

Reply:
Yeah, thanks.